system/Test/Mock/MockConnection.php

```diff
@@ -43,14 +43,14 @@ public function shouldReturn(string $method, $return)
 	 * Should automatically handle different connections for read/write
 	 * queries if needed.
 	 *
-	 * @param string  $sql
-	 * @param mixed   ...$binds
-	 * @param boolean $setEscapeFlags
-	 * @param string  $queryClass
+	 * @param mixed       $sql
+	 * @param mixed       ...$binds
+	 * @param boolean     $setEscapeFlags
+	 * @param string|null $queryClass
 	 *
 	 * @return BaseResult|Query|false
 	 */
-	public function query(string $sql, $binds = null, bool $setEscapeFlags = true, string $queryClass = 'CodeIgniter\\Database\\Query')
+	public function query($sql, $binds = null, bool $setEscapeFlags = true, ?string $queryClass = null)
```

Comment:
while the case is probably minimal, this is bc break,  ref https://3v4l.org/QES73